R/qualification-comparison-time-profile.R

```diff
@@ -166,11 +166,12 @@ addOutputToComparisonTimeProfile <- function(outputMapping, simulationDuration,
       molWeight = molWeight
     )
     observedValues <- observedValues[selectedObservedTimeValues]
+    observedResults$data <- observedResults$data[selectedObservedTimeValues,]
```

Comment:
`observedResults$data[selectedObservedTimeValues,]`
That comma at the end: correct or syntax error?

Reply:
It is correct, it is the R way of saying you apply it to all the columns (similar as the `:` in Matlab)

Reply:
ok